src/java/org/apache/cassandra/io/sstable/Component.java

```diff
@@ -34,7 +34,7 @@
  */
 public class Component
 {
-    public static final char separator = '-';
+    public static final char SEPARATOR = '-';
```

Comment:
This will break CNDB in the (brief?) time lapse between merging the CC PR and the CNDB PR. I guess that's ok.

Reply:
See https://github.com/riptano/cndb/pull/12715 and it needs a review :)